src/src.ruleset

```diff
@@ -8,6 +8,7 @@
     <Rule Id="CA1026" Action="None" />
     <Rule Id="CA1031" Action="None" />
     <Rule Id="CA1033" Action="None" />
+    <Rule Id="CA1305" Action="None" />
```

Comment:
This is a compiler warning for using CultureInfo in string.Format calls. The string interpolation I added above was causing an error. We don't need these errors.